### ISSUE TITLE
NullPointerException when attempting to search again after not finding seed

### ISSUE THREAD

**Issue by PixLSteam** [Leanny/SeedSearcher#5]
Version: [1.1RC](https://github.com/Leanny/SeedSearcher/releases/tag/1.1RC)
Accelerator used: GPU
Input: ![image](https://user-images.githubusercontent.com/21047644/73236509-e3236600-4192-11ea-9e05-b3dde7438b1e.png)
Error: 
![image](https://user-images.githubusercontent.com/21047644/73236586-31386980-4193-11ea-8fea-ae2788efe86a.png)
Stacktrace:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at SeedSearcherGui.SeedSearcherGui.CheckInput()
   at SeedSearcherGui.SeedSearcherGui.BT_Search_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
Note that it also changed the background color of one of the IVs for the 4IV (Day 4) Pokemon to red again, but pressing "Confirm IVs" changed it to white again. The error still occured. Restarting the application and entering the same values again allows you to search again.

**Comment by PixLSteam** [comment]
There's also some inputs where it can't find the seed, even with IV deviation 0 - 100. I was able to find the seed by skipping a raid but let me know if you're interested in the input I used.

**Comment by Leanny** [comment]
Yes, please send me the input, as well as the seed from a few frames later.

I will investigate the issue. So basically: you searched, it failed, you clicked on confirm ivs, you searched again?

**Comment by PixLSteam** [comment]
Yes. I just checked and the red background only appeared after clicking "Search" again. It also reset the corresponding Pokemon and ability, but not the nature or characteristic.

**Comment by PixLSteam** [comment]
I think this was the input where I didn't get a seed: ![image](https://user-images.githubusercontent.com/21047644/73250495-445d3080-41b7-11ea-99dc-0d2feda63bda.png)
Note that the screenshot is from an older version, but I tried getting the seed on 1.1 and still didn't get it. The new seed (skipped 1 raid) is CEB9F1F9CB428389

**Comment by Leanny** [comment]
If you use the seed and check stats for frame 3-5 (since you advanced by 1 frame 3-5 should be your input for the screen you sent me) 
Ability for day 5 should be Hustle, not Swarm
Nature for day 6 should be Timit, not Serious
So I guess this is the issue.

**Comment by PixLSteam** [comment]
Weird. I can imagine forgetting to set the correct ability but I don't know why I would select a totally different nature. Might try catching another one of those to verify.
Edit: Managed to catch both again and you're right, it's Hustle and Timid. Pretty weird that I managed to make 2 mistakes in the same search query while getting it right for countless other searches. Oh well, I'll need to be more careful from now on.
